script.js

```diff
@@ -0,0 +1,76 @@
+const button = document.getElementById('btn');
+const body = document.body;
+const menuField = document.querySelector('.wrap');
+const gameField = document.createElement('div');
+body.appendChild(gameField);
+isCardClick = false;
+
+
+//создание карты
+const addCardBox = () => {
+		const cardBox = document.createElement('div');
+		gameField.appendChild(cardBox);
+		cardBox.className = 'card_box';
+
+		const shirt = document.createElement('div');
+		shirt.innerHTML="<img src='images/card.png'>";
+		cardBox.appendChild(shirt);
+		shirt.className = 'shirt';
+
+		const faceCard = document.createElement('div');
+		faceCard.className = 'face';
+		let rand = Math.floor(Math.random() * 2);
+		console.log(rand);
+		if (rand == 0) {
+			faceCard.innerHTML = "<img src='images/cardBug.png'>";
+		} else if (rand == 1) {
```

Comment:
Codacy found an issue: [Use ===/!== to compare with true/false or Numbers](https://app.codacy.com/gh/WebHeroSchool/evagame/pullRequest?prid=5788637)